frontend/packages/dev-console/integration-tests/features/Knative/eventing-kamletes.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Creation of Kamelets via Event Source Creation
+    As a user, I should be able to install and use more camel connectors
+
+
+    Background:
+        Given user has installed OpenShift Serverless Operator
+        And user has created Knative Serving and Knative Eventing CR
+        And user has installed Red Hat Integration - Camel Operator
+        And user has created Integration Platform CR
+
+
+    @regression
+    Scenario: Kamelets on Search page
+        Given user is at Developer Perspective
+        When user goes to Search page
+        And user checks Kamlet checkbox
+        Then user will see aws-kinesis-source, aws-sqs-source, jira-source, salesforce-source, slack-source, telegram-source kamelets installed
```

Comment:
we may not need this , if kamelets is there should show up under eventSources which you already covered below

Reply:
Removed the scenario.